apps/core-angular-cli/src/app/app.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component } from '@angular/core';
-import '@clr/core/alert';
-import '@clr/core/button';
+import '@clr/core/alert/register.js';
```

Comment:
Does this really need `.js`?

Reply:
For Angular I don't think so. I think the underlying Webpack will search and resolve for `index.js` or `register.js`. Non webpack would need `.js`.

Reply:
Yes, all our imports use `.js`. See #4416